tests/unit/RoleCredentials.js

```diff
@@ -86,4 +93,21 @@ describe('Credentials Manager', () => {
             }, retryTimeout);
         });
     });
+
+    it('should handle non arsenal errors on refresh', function test(done) {
+        this.timeout(10000);
+        // wait for an extra second after timeout to ensure credentials
+        // have expired
+        const retryTimeout = (roleCredentials.expiration - Date.now()) +
+            1000;
+        return setTimeout(() => {
+            assert(roleCredentials.expired === false,
+                'expected credentials to expire');
```

Comment:
do you mean the opposite "not to expire"? You may also use `assert.strictEqual`

Reply:
Ah, yes.